### DISCUSSION TITLE
EV3 MicroPython - Stuck during boot

### DISCUSSION THREAD

**Discussion by DylanBPY** [pybricks/support#1292] [Q&A] [answered]
This is a rather weird issue to have. I have a Lego Mindstorms EV3 running MicroPython. I can boot it fine one time, once I shut it down and start it again, it doesn't boot. It gets stuck at the EV3 Dev page. So then I have to take my bot apart and take the batteries out and put them back in. Once I reboot it after that, it works fine.

I have tried a reset using this [Guide](https://www.lego.com/en-sg/service/help/digital/electronics-and-robotics/mindstorms-ev3/ev3-p-brick-reset-and-update-blt864c013461ef63ac) and updating the firmware to the latest version. There seems to be nothing about this online and I am honestly out of ideas as to what this could be.

A more basic timeline:
Boots fine once -> Restart -> Doesn't boot -> Take out batteries and put them back in -> Works Fine -> Repeats

**Comment by dlech** [comment]
We've been getting reports lately that Etcher is broken. You can flash with a different program or unzip the file and flash the .img file using Etcher instead of flashing the .zip file.

**Reply by dlech** [reply, marked as answer]
Also, if you have a microSD card that is bigger than 32GB, that could be the issue.

**Reply by DylanBPY** [reply]
My card is 64GB so I will find my other 32gb and try that

**Reply by DylanBPY** [reply]
I used a 32GB SD Card and Etcher 1.5.26 and it works now
